pandas/tests/apply/test_frame_apply.py

```diff
@@ -1643,7 +1647,7 @@ def test_apply_no_suffix_index():
 
 
 def test_apply_raw_returns_string():
-    # https://github.com/pandas-dev/pandas/issues/35940
+    # GH 35940
```

Comment:
I personally don't find this an improvement. I know we are inconsistent about it, but if there is a full link, it's much easier to go to the issue

Reply:
this is how we do for every other comment about the issue number

if we want to require a full link it's possible but would need a precommit hook

Reply:
We don't do that for "every other comment". We currently use the full link in 400 cases. 

Not every detail needs to be controlled with a pre-commit hook

Reply:
and how many w/o a link? i don't know but i would say it's way higher

we have to have a standard and then enforce it

we simply cannot track stylistic things any other way or involve personal preference here (once we agree on a standard)

Reply:
@jorisvandenbossche - Thanks for the feedback here, I didn't realize there wasn't a consensus. I actually agree on the preference for a link (but also more strongly prefer consistency). I've reverted these changes.

Assuming we can get a consensus, if things were made consistent and we updated the dev docs, it seems to me that consistency would become the norm and not require enforcement after a short amount of time. If it doesn't turn out to be the case and becomes a hassle, we could then implement a pre-commit hook.